src/components/button/Button.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Kindly prefer not to `disable` the linter errors, but rather solve the error. This will help to avoid unwanted bugs or semantic errors in the code. So, I suggest you to remove this line and fix that linter error.

Happy Coding 👍